src/Resolver/OrganizationIdResolverInterface.php

```diff
@@ -1,5 +1,12 @@
 <?php
 
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
 declare(strict_types=1);
 
 /*
```

Comment:
Could you check, why we have doubled OS header? And I would also ask you for checking whether it's the only place with it.

Reply:
OrganizationIdResolverInterface OS_Header double appearance has been fixed.